utreexo-validation-bip.md

```diff
@@ -0,0 +1,342 @@
+```
+BIP: TBD
+Layer: Peer Services
+Title: Utreexo - Validation Layer
+Authors: Tadge Dryja <rx@awsomnet.org>
+         Calvin Kim <bip@calvinkim.info>
+         Davidson Souza <bip@dlsouza.dev>
+Comments-URI: TBD
+Status: Draft
+Type: Specification
+Created: 2023-10-01
+License: BSD-3-Clause
+Depends: BIP-???? (Utreexo Accumulator Specification)
+```
+
+## Abstract
+
+This BIP defines the rules for validating blocks and transactions using the
+Utreexo accumulator. It is important to note that this BIP does not define the
+Utreexo accumulator itself, for that see BIP-????. This document is only concerned with
+the general rules for validating blocks and transactions using the Utreexo,
+so that all Utreexo nodes can stay in consensus with one another.
+
+## Motivation
+
+The Bitcoin network is composed of a set of nodes that validate blocks and
+transactions as they are received. These nodes need to keep track of the current state of the network in order to
+fulfill their role. Most importantly, they must maintain a record of all coins that
+have been created but not yet spent—a collection known as the UTXO set.
+
+This set is typically stored in a database that must be accessed frequently and cannot
+be pruned. As a result, the cost of running a node is directly tied to the size
+of the UTXO set. Since it can grow indefinitely—bounded only by block size—it represents a
+long-term scalability concern.
+
+Utreexo is a dynamic accumulator that enables the UTXO set to be represented in just a few kilobytes,
+by requiring peers to provide additional proof data to verify the inclusion of a UTXO in the
+accumulator. This allows for the construction of extremely lightweight nodes capable of performing
+the same validation as a full node, without the need to store the entire UTXO set.
```

Comment:
The preceding 3 paragraphs seem to be duplicates of the accumulator BIP that this BIP requires. Can perhaps remove them or refer to the accumulator BIP motivation.